github/commands/pull_request.py

```diff
@@ -101,17 +101,20 @@ def fetch_and_checkout_pr(self, branch_name=None):
             self.pr["head"]["ref"]
         )
 
-        if branch_name:
-            self.window.status_message("Creating local branch for PR...")
-            self.git(
-                "branch",
-                branch_name,
-                self.pr["head"]["sha"]
-            )
+        if not branch_name:
```

Comment:
This change is not ok since it breaks the "Checkout detached" feature. I mean `branch_name` can be `None` intentionally.